frog/imports/ui/EditOperators.jsx

```diff
@@ -60,7 +60,7 @@ const OperatorFormComponent = (props) => {
       type: "string",
       title: 'Source node (activity)',
       enum: map(props.activities, (k, v) => k._id),
-      enumNames: props.activities.map(k => k.data.name)
+      enumNames: props.activities.map(k => `${k.data.name} (${k.activity_type})`)
```

Comment:
A bit of a different issue, but didn't want to make separate pull request for this :)